ADR/nnnn-trusted-task-model.md

```diff
@@ -0,0 +1,245 @@
+# NN. Trusted Tasks model after build-definitions decentralization
+
+Date: 2025-09-30
+
+## Status
+
+Proposed
+
```

Comment:
I want to cross-link this the "partner tasks" ADR from way back when. De-centralizing build-definitions is essentially a new way to "do partner tasks".

Reply:
I see what you mean. I think I would still put the replaced-by link in the partner task one, just so that people can find their way here when they stumble on that one.